### PR TITLE
Metrolist dev

### PR DESCRIPTION
**Pull request by padora-cob**
Merge new MetroList features into develop

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,6 +6,8 @@ branches:
   only:
     - develop
     - master
+    - metrolist_dev
+    - metrolist_ci
```
Comment (padora-cob) [inline .travis.yml:10]: Should I remove these: 

```suggestion
    - metrolist_dev
    - metrolist_ci
```

File: .travis.yml [hunk 2]
```diff
@@ -116,3 +118,13 @@ deploy:
...
+  - provider: script
+    skip_cleanup: true
+    script: bash $TRAVIS_BUILD_DIR/scripts/deploy/travis_deploy.sh $TRAVIS_BRANCH
+    on:
+      branch: metrolist_dev
+  - provider: script
+    skip_cleanup: true
+    script: bash $TRAVIS_BUILD_DIR/scripts/deploy/travis_deploy.sh $TRAVIS_BRANCH
+    on:
+      branch: metrolist_ci
```
Comment (padora-cob) [inline .travis.yml:130]: Should I remove these:

File: config/default/core.extension.yml [hunk 3]
```diff
@@ -62,20 +65,23 @@ module:
   ckeditor: 0
   color: 0
   config: 0
+  config_devel: 0
```
Comment (davidrkupton) [inline config/default/core.extension.yml:68]: this should be off - delete this line.

File: config/default/core.extension.yml [hunk 4]
```diff
@@ -62,20 +65,23 @@ module:
...
   contextual: 0
+  contextual_range_filter: 0
   ctools: 0
   date_popup: 0
   date_recur: 0
   date_recur_modular: 0
   datetime: 0
   datetime_range: 0
   dblog: 0
+  devel: 0
```
Comment (davidrkupton) [inline config/default/core.extension.yml:84]: this should be off - delete this line. full list to remove:
- config_devel
- devel
- stage_file_proxy
- twig_xdebug

Comment (padora-cob) [reply config/default/core.extension.yml]: Removed the dev modules, thanks David!

File: .travis.yml (hunk 1 above)
Comment (padora-cob) [reply .travis.yml]: Removed

File: config/default/views.view.events_and_notices.yml [hunk 5]
```diff
@@ -23,6 +23,7 @@ dependencies:
     - node
```
Comment (davidrkupton) [inline config/default/views.view.events_and_notices.yml:23]: Unless you did actualy make changes in this file (and I dont think you did...), I would exclude it.

File: config/default/views.view.watchdog.yml [hunk 6]
```diff
@@ -1,715 +0,0 @@
-uuid: 5f9e5d57-b6d0-4450-8e39-8b8fc3dfbbbe
```
Comment (davidrkupton) [inline config/default/views.view.watchdog.yml:1]: I think this file should be left in the repo.

File: config/default/views.view.events_and_notices.yml (hunk 5 above)
Comment (padora-cob) [reply config/default/views.view.events_and_notices.yml:23]: Ah, yes we did make changes. Added a new Views Display Block that we can feed IDs to as contextual filters and get all of there news and events as one group.

File: config/default/views.view.watchdog.yml (hunk 6 above)
Comment (padora-cob) [reply config/default/views.view.watchdog.yml]: Just added it back, thanks for catching that.

File: scripts/.config.yml [hunk 7]
```diff
@@ -63,6 +63,34 @@ build:
         # Set to 'true' or 'false'.  False means no configs will be imported during build/deploy
         # Note: if the commit is a hotfix, then configs wont be imported for expediency.
         sync: "true"
+    metrolist_dev:
```
Comment (davidrkupton) [inline scripts/.config.yml:66]: Alex: yopu can remove these too

Comment (padora-cob) [reply scripts/.config.yml]: The MetroList development ENV's have been removed.

**Pull request merged**